extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -697,10 +697,37 @@ static public function onArticleDelete( $article, &$user, &$reason, &$error ) {
 	}
 
 	/**
+	 * Adjusting Recent Changes for Wall
+	 * Change RC db key to affect Watchlist behavior
+	 * @author TK-999 <https://github.com/TK-999>
+	 * @param RecentChange $rc
+	 * @return Bool true because it is a hook
+	 */
+	static public function onRecentChangeBeforeSave( RecentChange &$rc ) {
+		$wg = F::app()->wg;
+
+		if ( MWNamespace::isTalk( $rc->getAttribute( 'rc_namespace' ) ) &&
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "rc_namespace" 25 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)